features/steps/print_steps.py

```diff
@@ -20,9 +21,11 @@ def wait_for_print_files(context):
                 context.produced_print_file_time = datetime.utcnow()
                 context.print_file_production_run_time = context.produced_print_file_time \
                     - context.action_rule_trigger_time
-                # TODO nicer way to print within behave
-                print(f'Time from action rule trigger to all print files produced: '
-                      f'[{str(context.print_file_production_run_time)}]\n')
+                time_taken_metric = json.dumps({
+                    'event': 'Time from action rule trigger to all print files produced',
```

Comment:
Can I be a real fusspot and suggest that this might be better as `event_description` and then we use short identifiers for each event 'type' like `ACTION_RULE_TO_PRINT` to make it easier to graph certain things?